packages/client/src/pages/location/location.tsx

```diff
@@ -63,6 +64,7 @@ const LocationRoutes = () => {
           <LoadingView fullScreen animated title={t('common:loader.loadingApp')} titleClassname="text-black" />
         </div>
       )}
+      {createPortal(<canvas id="engine-renderer-canvas" tabIndex={1} ref={canvasRef}></canvas>, document.body)}
```

Comment:
leaving a note here that this can likely be temporary as I just have to fix input handling since a lot of our input handling relied on the canvas being in document.body